DarkFlippers/unleashed-firmware
# BadUSB: Add Finnish keyboard layout

# What's new

- Added Finnish keyboard layout for BadUSB (generated using [flipperzero_badusb_kl](https://github.com/dummy-decoy/flipperzero_badusb_kl))

# Verification 

- Activate the Finnish keyboard layout on your PC
- Open the BadUSB application on Flipper and choose a script that enters some special characters like `$()`
- Connect Flipper to the PC using a USB cable
- Select the Finnish keyboard layout on Flipper
- Run the script
- Verify that the correct keystrokes have been inserted

# Checklist (For Reviewer)
